web/src/emoji_picker.js

```diff
@@ -789,10 +789,11 @@ function register_click_handlers() {
         );
     });
 
-    $("body").on("click", "#set-user-status-modal #selected_emoji", (e) => {
-        e.preventDefault();
```

Comment:
Removed `e.preventDefault()` as it prevented tabbing into further options by disabling the default action of the tab key.